### ISSUE TITLE
Support ability to hide `amp-social-share` in favor of native sharing

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#14188]

**Comment by aghassemi** [comment]
This is a request for cases when a platform embedding AMP (e.g. some native app) has its own share UI.

**Comment by caroqliu** [comment]
@nainar Any sense if this is distinct from [`type=system`? ](https://amp.dev/documentation/components/amp-social-share/#pre-configured-providers)

Otherwise this should be reasonable to close.
